packages/python-peid/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-peid
+_pyname=peid
+pkgver=1.2.9
+pkgrel=1
+pkgdesc='Python implementation of the Packed Executable iDentifier (PEiD)'
+arch=('any')
+grouos=('blackarch' 'blackarch-binary' 'blackarch-reversing')
+license=('GPL3')
+depends=('python' 'python-pefile' 'python-pip')
+makedepends=('python-setuptools')
```

Comment:
no need of it if you don't use it

Reply:
I have no idea if `python-setuptools` really needs to be removed. But in the project.toml it seems to be a build requirement in https://github.com/packing-box/peid/blob/main/pyproject.toml#L2

Reply:
You can test with and without it to see if it's really needed. https://gist.github.com/noraj/5045be32490f1161161e16fc5820066c#testing

Reply:
I've realized it does not need `python-setuptools` but instead `python-build` is required now. Thanks for making me aware about that.